docs/quickstarts/openshift-ai-rhods-related-documentation/openshift-ai-rhods-related-documentation.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: console.openshift.io/v1
+kind: QuickStarts
+metadata:
+  name: openshift-related-documentation
```

Comment:
This name does not match the name in the metadata. It should be `openshift-ai-rhods-related-documentation`